libs/pageserver_api/src/controller_api.rs

```diff
@@ -101,6 +101,19 @@ pub struct TenantDescribeResponse {
     pub config: TenantConfig,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct NodeShardResponse {
+    pub node_id: NodeId,
+    pub shards: Vec<NodeShard>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct NodeShard {
+    pub tenant_shard_id: TenantShardId,
+    pub is_observed_secondary: Option<bool>,
```

Comment:
Why `Option<bool>` rather than just bool?

Reply:
None == the tenant doesn't exist on this node. I have a commit but it didn't get pushed...?

Reply:
None == the tenant doesn't exist on this node. I have a commit to add the comments but it didn't get pushed...?

Reply:
Ah, so we're saying Some(false) means "this shard is attached", but None means "this shard does not appear in the intent state" (but it must appear in the observed state or we wouldn't have constructed a NodeShard).

This is a bit subtle, but okay - the CLI output is still easy to understand.

Reply:
I couldn't find a better naming for that field :( anyways, the updated comments:
```
    /// Whether the shard is observed secondary on a specific node. True = yes, False = no, None = not on this node.
    pub is_observed_secondary: Option<bool>,
```